cypress/support/fragments/inventory/inventoryItem/itemView.js

```diff
@@ -5,7 +5,8 @@ import {
   HTML,
   including,
   TextField,
-  MultiColumnList
+  MultiColumnList, 
+  PaneHeader
```

Comment:
'PaneHeader' is defined but never used. Allowed unused vars must match /React/u

Reply:
fixed